package.json

```diff
@@ -1,41 +1,39 @@
 {
   "name": "typesync",
   "version": "0.13.4",
+  "type": "module",
   "description": "Install missing TypeScript typings for your dependencies.",
   "engines": {
-    "node": ">=16.0.0"
+    "node": "^18.20.0 || ^20.10.0 || >=22.0.0"
   },
   "files": [
-    "lib",
+    "dist",
     "bin"
   ],
   "bin": {
     "typesync": "./bin/typesync"
   },
-  "main": "lib/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    }
+  },
```

Comment:
Just double-checking—we actually want to expose this, right?

Reply:
Yes